boot/seal.go

```diff
@@ -592,6 +592,10 @@ func runModeBootChains(rbl, bl bootloader.Bootloader, model *asserts.Model, mode
 // hashes from modeenv plus it returns separately the last BootFile
 // which is for the kernel.
 func buildBootAssets(bootFiles []bootloader.BootFile, modeenv *Modeenv) (assets []bootAsset, kernel bootloader.BootFile, err error) {
+	if len(bootFiles) == 0 {
+		// useful in testing, when mocking is insufficient
+		return nil, bootloader.BootFile{}, fmt.Errorf("internal error: no boot files")
```

Comment:
maybe it doesn't matter because it will show up only in tests, so there's context. I'm just slightly worried that as a message in doesn't make too clear where it is coming from. Though at least atm we don't have any ohter "no boot files" error